extensions/wikia/JWPlayerTag/scripts/jwplayertag.js

```diff
@@ -0,0 +1,45 @@
+require(['jquery'], function ($) {
+	var parserTagSelector = '.jwplayer-in-article-video .jwplayer-container',
+		jwVideoDataUrl = 'https://cdn.jwplayer.com/v2/media/';
+
+	function init() {
+		var parserTags = document.querySelectorAll(parserTagSelector);
+
+		Array.prototype.slice.call(parserTags).forEach(function (each) {
+			// We check this to avoid errors in places where we don't load JW
+			// e.g. Monobook
+			if (window.WikiaJWPlayer) {
+				setupPlayer(each);
+			}
+		});
+	}
+
+	function getPlayerSetup(jwVideoData) {
+		return {
+			autoplay: {
```

Comment:
Let's pull in dev branch, cause this is no longer correct. `autoplay` is now boolean